Bottom: ee0ead8c5bfdb444ae4a97562c1ecbf716a2c9f6
Top:    82bb08138c6d436da67ee5ec6d655691df613629
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-10 19:30:34 +0000

Refresh of attempt-to-fix-slowness

---

diff --git a/src/polygon1.c b/src/polygon1.c
index ca5ec95..a0a0abb 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -944,7 +944,7 @@ cntr_in_M_PLINE (PLINE * poly, PLINE * outfst, BOOLp test)
     if (poly_ContourInContour (outer, poly)) {
       for (curc = outer->next; curc != NULL; curc = curc->next)
         if (poly_ContourInContour (curc, poly)) {
-          /* it's inside a hole in the smallest polygon 
+          /* it's inside a hole in the smallest polygon
            * no need to check the other polygons */
           heap_destroy (&heap);
           return FALSE;
